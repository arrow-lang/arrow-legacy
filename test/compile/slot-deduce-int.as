let x: int8 = 0;
let val = x; // should be `int8`
