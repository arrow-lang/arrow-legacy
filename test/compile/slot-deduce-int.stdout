@x = internal constant i8 0
@val = internal global i8 0

define internal void @slot-deduce-int..init() {
  %1 = load i8, i8* @x
  store i8 %1, i8* @val
  ret void
}

define i32 @main(i32, i8**, i8**) {
  call void @slot-deduce-int..init()
  ret i32 0
}
